crypto/src/main/java/net/corda/v5/crypto/CompositeKeyNodeAndWeight.java

```diff
@@ -0,0 +1,29 @@
+package net.corda.v5.crypto;
+import org.jetbrains.annotations.NotNull;
+
+import java.security.PublicKey;
+
+
+/**
+ * A simple data class for passing keys and weights into CompositeKeyGenerator
+ *
+ * @param node A public key
+ * @param weight The weight for that key, must be greater than zero.
+ */
+public final class CompositeKeyNodeAndWeight {
```

Comment:
This used to be a data class. It will need that we define its `equals` `hashCode` and `toString`

Reply:
done